README.md

```diff
@@ -101,6 +101,31 @@ In addition to basic error notification, typical operational warnings are genera
      --workflow ingestion_pipeline.conf \
      --rerun 2022-01-01
    ```
+   
+## Building Pramen to suite your environment
+
+Pramen is released a thin JAR dependencies. When running on a specific environment you might want to tune the pipeline
```

Comment:
Not sure, if I get this correct, but looks like this should be applied:
```diff
- Pramen is released a thin JAR dependencies
+ Pramen is released as a thin JAR dependencies
```

Reply:
Agree, thanks for spotting!